plugins/nf-lamin/src/main/nextflow/lamin/model/RunStatus.groovy

```diff
@@ -0,0 +1,36 @@
+package nextflow.lamin.model
+
+import groovy.transform.CompileStatic
+
+/**
+ * Status codes for workflow runs in Lamin
```

Comment:
Can you add this link here to the comment?

https://github.com/laminlabs/lamindb/blob/768c66c6347e1a098ffd0b9e03f77685c47f679c/lamindb/models/run.py#L323

Reply:
Check!